### PR TITLE
Add maven package

### PR DESCRIPTION
**Pull request by ImpLarsBruemmer**

### CODE REVIEW

File: functions.js [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-    let data = {};
-    data.id = e.id;
-    data.version = e.metadata.container.tags[0];
-    data.updated_at = e.updated_at;
-    return data;
-  });
+  let mappedData;
+  if (packageType === "maven") {
+    mappedData = json.map(e => mapMavenVersionData(e));
+  } else {
```
Comment (ImpThomasHein) [inline functions.js:9]: maybe you just do the handling here for docker images, npm packages might also work differently

File: functions.js [hunk 2]
```diff
@@ -36,22 +35,72 @@ function sortByDateDescending(a, b) {
...
+  }
+
+  let versionsSplit = versionNames.split(',').map(e => e.trim());
+  if (versionsSplit.length < 1 || (versionsSplit.length === 1 && versionsSplit[0] === "")) {
+    return undefined;
+  }
+
+  if (packageType === "maven") {
+    return filterMavenVersionsByName(json, versionsSplit);
+  } else {
```
Comment (ImpThomasHein) [inline functions.js:68]: also here, use the specific packageType

**Review by ImpThomasHein — CHANGES_REQUESTED** [review]

File: functions.js (hunk 1 above)
Comment (ImpLarsBruemmer) [reply functions.js:9]: I added docker and npm handling

File: functions.js (hunk 2 above)
Comment (ImpLarsBruemmer) [reply functions.js:68]: container is the default one that is why I use the else here

**Review by ImpThomasHein — APPROVED** [review]

**Pull request merged**